ClinGen/clincoded
# Connects to #1561. Adding new users for R15.

**Steps to test:**
1. Go to ^/users/.
2. Enter either **amanda.thomas@aruplab.com** or **y.cindy.si@gmail.com** in the **_Filter table by_** search box at the upper right hand corner.
3. Expect to see the filtered entries being returned for the users and their associated affiliation IDs.
